apps/aechannel/src/aesc_session_noise.erl

```diff
@@ -193,7 +217,7 @@ handle_info({'DOWN', Ref, process, Pid, _Reason},
             #st{parent_mon_ref=Ref, parent=Pid, econn=EConn}=St) ->
     lager:debug("Got DOWN from parent (~p)", [Pid]),
     close_econn(EConn),
-    {stop, normal, St};
+    {stop, shutdown, St};
```

Comment:
I expect this change to change the `terminate` reason though without any functional or logging change. Ok.